corehq/motech/dhis2/entities_helpers.py

```diff
@@ -197,22 +199,13 @@ def update_tracked_entity_instance(
     requests, tracked_entity, etag, case_trigger_info, case_config,
     attempt=1
 ):
-    case_updates = {}
-    for attr_id, value_source_config in case_config['attributes'].items():
-        value, case_update = get_or_generate_value(
-            requests, attr_id, value_source_config, case_trigger_info
-        )
-        set_te_attr(tracked_entity["attributes"], attr_id, value)
-        case_updates.update(case_update)
-    enrollments_with_new_events = get_enrollments(
+    tracked_entity, case_updates = _build_tracked_entity(
+        requests,
+        tracked_entity,
         case_trigger_info,
         case_config,
+        True
```

Comment:
This is a nitpick, so don't feel you _have_ to change this: When passing more than two parameters to a method, I always try to use keyword arguments, it makes it easier to read. Especially in this case, one has to go to the function definition to see what this boolean means. Here might be an excellent place to use a keyword argument :smile:

Reply:
That makes sense, would definitely help with readability. Thanks, I'll definitely keep that in mind!